modules/rank-eval/build.gradle

```diff
@@ -0,0 +1,35 @@
+/*
+ * Licensed to Elasticsearch under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+esplugin {
+  description 'The Rank Eval module adds APIs to evaluate ranking quality.'
+  classname 'org.elasticsearch.index.rankeval.RankEvalPlugin'
+}
+
+dependencies {
+  compile "com.github.spullara.mustache.java:compiler:0.9.1"
+  compile project(':modules:lang-mustache')
```

Comment:
We can't have modules depending on other modules. This means we will actually load two copies, because what is here now will copy the lang-mustache jars into this module, and this module and lang-mustache will have their own classloaders.

Why do you need a dependency directly on mustache scripting? Should it just be on script templates? Perhaps #16314 is what we need first?